apps/mobile/.env.development

```
@@ -2,19 +2,20 @@
 #/            public-key encryption for .env files          /
 #/       [how it works](https://dotenvx.com/encryption)     /
 #/----------------------------------------------------------/
-# .env.development.keys
-DOTENV_PUBLIC_KEY_DEVELOPMENT=0345437ff2783e4d7cd0572c2dd148242b9c9025a1f51faa719b2882fa6d4088e2
+DOTENV_PUBLIC_KEY_DEVELOPMENT="0345437ff2783e4d7cd0572c2dd148242b9c9025a1f51faa719b2882fa6d4088e2"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)